### PR TITLE
New office code

### LINKED ISSUE #7: Update re: new office key codes

### PR DESCRIPTION
**Pull request by csteinert**
Updates wording around Columbus office passcodes.

Closes #7

### CODE REVIEW

File: day-to-day.md [hunk 1]
```diff
@@ -129,21 +135,12 @@ Definition of a good meeting:
 
 The entire team meets once a week for lunch (paid for by Base Two) to go over the state of the company. The meeting typically lasts about 1.5 hours. The agenda for each meeting is prepared that morning (in a shared Google Doc) and all team members are welcome to contribute. We cover any combination of the following (incomplete) list of topics:
 
-- Financials
-	- Account balances
-	- Invoices/billing
-	- Profitability
-	- Hours
+- Financials - Account balances - Invoices/billing - Profitability - Hours
```
Comment (reeddunkle) [inline day-to-day.md]: ❓ I just want to make sure that the one-line formatting here and below is intentional?

**Review by reeddunkle — APPROVED** [review]
:nut_and_bolt: :night_with_stars: Thanks for doing this. Nice work. I left a question below to confirm the refactor you made was intentional and not just auto-formatting from the linter.

File: day-to-day.md (hunk 1 above)
Comment (dmiller-nj) [inline day-to-day.md]: It looks like these (and two groupings below) sub-bullets got mangled in the re-format.

**Review by dmiller-nj — APPROVED** [review]
LGTM! :notebook_with_decorative_cover: :night_with_stars:

File: day-to-day.md (hunk 1 above)
Comment (csteinert) [reply day-to-day.md]: Definitely some kind of weird auto formatting. Will fix. Thank you!

**Pull request merged**